### PR TITLE
doc: update readme with guided_json and other extra parameters

### PR DESCRIPTION
**Pull request by larme**

### CODE REVIEW

File: llama2-7b-chat/README.md [hunk 1]
```diff
@@ -110,6 +110,37 @@ for chunk in chat_completion:
     print(chunk.choices[0].delta.content or "", end="")
 ```
 
+These OpenAI-compatible endpoints also support [vLLM extra parameters](https://docs.vllm.ai/en/latest/serving/openai_compatible_server.html#extra-parameters). For example, you can force the chat completion output a JSON object by using the `guided_json` parameters:
```
Comment (Sherlock113) [inline llama2-7b-chat/README.md:113]: `guided_json` means `json_schema` here? Users don't need to set it in the `client.chat.completions.create` function?

Comment (larme) [reply llama2-7b-chat/README.md:113]: Yes, `guided_json` just forces the output to obey the JSON schema. User passed `guided_json` to extra_body. I missed that in my code, just updated.

File: solar-10.7b-instruct/README.md [hunk 2]
```diff
@@ -99,6 +99,38 @@ for chunk in chat_completion:
...
+
+json_schema = {
+    "type": "object",
+    "properties": {
+        "city": {"type": "string"}
+    }
+}
+
+chat_completion = client.chat.completions.create(
+    model="mistralai/Mistral-7B-Instruct-v0.2",
```
Comment (Sherlock113) [inline solar-10.7b-instruct/README.md:120]:
```suggestion
    model="upstage/SOLAR-10.7B-Instruct-v1.0",
```

File: solar-10.7b-instruct/README.md [hunk 3]
```diff
@@ -99,6 +99,38 @@ for chunk in chat_completion:
...
+            "role": "user",
+            "content": "What is the capital of France?"
+        }
+    ],
+    extra_body=dict(guided_json=json_schema),
+)
+print(chat_completion.choices[0].message.content)  # will return something like: {"city": "Paris"}
+```
+
+All supported extra parameters are listed at [vLLM documentation](https://docs.vllm.ai/en/latest/serving/openai_compatible_server.html#extra-parameters).
```
Comment (Sherlock113) [inline solar-10.7b-instruct/README.md:132]:
```suggestion
All supported extra parameters are listed in [vLLM documentation](https://docs.vllm.ai/en/latest/serving/openai_compatible_server.html#extra-parameters).
```

File: mixtral-8x7b-instruct/README.md [hunk 4]
```diff
@@ -103,6 +103,38 @@ for chunk in chat_completion:
...
+            "role": "user",
+            "content": "What is the capital of France?"
+        }
+    ],
+    extra_body=dict(guided_json=json_schema),
+)
+print(chat_completion.choices[0].message.content)  # will return something like: {"city": "Paris"}
+```
+
+All supported extra parameters are listed at [vLLM documentation](https://docs.vllm.ai/en/latest/serving/openai_compatible_server.html#extra-parameters).
```
Comment (Sherlock113) [inline mixtral-8x7b-instruct/README.md:136]:
```suggestion
All supported extra parameters are listed in [vLLM documentation](https://docs.vllm.ai/en/latest/serving/openai_compatible_server.html#extra-parameters).
```

File: mistral-7b-instruct/README.md [hunk 5]
```diff
@@ -103,6 +103,38 @@ for chunk in chat_completion:
...
+            "role": "user",
+            "content": "What is the capital of France?"
+        }
+    ],
+    extra_body=dict(guided_json=json_schema),
+)
+print(chat_completion.choices[0].message.content)  # will return something like: {"city": "Paris"}
+```
+
+All supported extra parameters are listed at [vLLM documentation](https://docs.vllm.ai/en/latest/serving/openai_compatible_server.html#extra-parameters).
```
Comment (Sherlock113) [inline mistral-7b-instruct/README.md:136]:
```suggestion
All supported extra parameters are listed in [vLLM documentation](https://docs.vllm.ai/en/latest/serving/openai_compatible_server.html#extra-parameters).
```

File: llama2-7b-chat/README.md [hunk 6]
```diff
@@ -110,6 +110,38 @@ for chunk in chat_completion:
...
+            "role": "user",
+            "content": "What is the capital of France?"
+        }
+    ],
+    extra_body=dict(guided_json=json_schema),
+)
+print(chat_completion.choices[0].message.content)  # will return something like: {"city": "Paris"}
+```
+
+All supported extra parameters are listed at [vLLM documentation](https://docs.vllm.ai/en/latest/serving/openai_compatible_server.html#extra-parameters).
```
Comment (Sherlock113) [inline llama2-7b-chat/README.md:143]:
```suggestion
All supported extra parameters are listed in [vLLM documentation](https://docs.vllm.ai/en/latest/serving/openai_compatible_server.html#extra-parameters).
```

**Pull request merged**